src/test/unit/services/persisters/s3Persister.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const chai = require('chai');
+const sinon = require('sinon');
+const sinonChai = require("sinon-chai");
+chai.should();
+chai.use(sinonChai);
+
+describe('[UNIT] s3Persister', function() {
+  const s3Persister = require('../../../../services/persisters/s3Persister');
+  const UnitTest = require('../../unitTest');
+
+  UnitTest.Setup({
```

Comment:
See my comment about this same kind of thing in the integration test for this persister.
Essentially, change lines 13-23 to `UnitTest.Setup();`